AmrVisTool.cpp

```diff
@@ -267,8 +275,8 @@ int main(int argc, char *argv[]) {
 	dspArray[nPlots] = new amrex::DataServices(comlineFileName, fileType);
       }  // end for(nPlots...)
 
-      for(int nPlots(0); nPlots < AVGlobals::GetFileCount(); ++nPlots) {
-        if(amrex::ParallelDescriptor::IOProcessor()) {
+      if(amrex::ParallelDescriptor::IOProcessor()) {
```

Comment:
On Slack by @kngott:
> If I’m not mistaken, it was the buggy first attempt to parallelize that needed some serious testing.
> Just putting together and deleting isn’t the move.

Reply:
Maybe we should rename that branch or add some form of docs. I don't have powers there, but let's rebase the branch against main and call it feature-parallel-profvis or so.
Otherwise too confusing for a mainline branch.

Reply:
If you were looking to clean up a bit, renaming the branch is probably the way to go for now.